config/config.go

```diff
@@ -47,6 +49,17 @@ func NewDefaultConfig() *AgentConfig {
 				JwksURL:            "",
 				JwksUpdateInterval: 0,
 			},
+			CORS: CORSConfig{
+				// If allowedOrigins is nil or empty, value is set to ["*"].
+				AllowedOrigins: nil,
```

Comment:
Looks like this is defaulting to wide-open CORS support. We need to default to disabled then enable only if explicitly configured.

Reply:
agree with this comment

Reply:
Discussed offline, it's OK to comment it out and give an idea to the user, CORS is not enabled. It will work as expected without `cors` before implementing it how it was working.